assets/js/vue/FileManager.vue

```diff
@@ -37,9 +40,16 @@ export default {
         // Assigning it to the global variable of this class so that functions outside the export can use it
         datasetSubmissionId = this.datasetSubId;
     },
+
+    methods: {
+        onErrorOccurred: function(e) {
+            e.errorText = 'Cannot delete folders';
```

Comment:
Placeholder code, not sure how to call this method from the reject() in the Promise methods below when there is an exception